src/timezone.ts

```diff
@@ -0,0 +1,39 @@
+import { Message } from 'discord.js';
+import * as moment from 'moment-timezone';
+
+import config from './config';
+
+const { defaultEmbed, TIMEZONES } = config;
+
+export const timezone = async (message: Message) => {
```

Comment:
what do you think of adding some comments on this function to explain a bit the feature, like "This function is called for each "message" event and checks if it contains a timezone value. If it does it displays a embed message with alternative timezones to facilitate conversion"

Reply:
Yep good idea 👍